src/package.xml

```diff
@@ -2331,6 +2332,7 @@
         <members>stgHelpRDFieldMapRDField</members>
         <members>stgHelpRDLastRun</members>
         <members>stgHelpRDMaxDonations</members>
+        <members>stgHelpRDInstallmentCreateOptionMessage</members>
```

Comment:
The new field and custom label are not ordered in `package.xml`. Running the `cci flow` will update it.